src/test/java/commands/DeleteDeadlineCommandTest.java

```diff
@@ -0,0 +1,69 @@
+package commands;
+
+import author.AuthorList;
+import exceptions.TantouException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import ui.Ui;
+
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class DeleteDeadlineCommandTest {
+    private final PrintStream standardOut = System.out;
+    private AuthorList authorList;
+    private Ui ui;
+    private AddDeadlineCommand commandUnderTest;
+    private DeleteDeadlineCommand secondCommandUnderTest;
+
+    @BeforeEach
+    public void setUp() {
+        authorList = new AuthorList();
+        ui = new Ui();
+    }
+
+    @Test
+    public void deleteDeadlineCommand_deleteSingleDeadline_deadlineMatchAuthorCountOne() {
+        try {
+            commandUnderTest = new AddDeadlineCommand("add -a \"Gege Akutami\" " +
+                    "-m \"Jujutsu Kaisen\" -d \"September 29\"");
+            commandUnderTest.execute(ui, authorList);
+            secondCommandUnderTest = new DeleteDeadlineCommand("delete -a \"Gege Akutami\"" +
+                    " -m \"Jujutsu Kaisen\" -d \"September 29\"");
```

Comment:
Perhaps it isn't necessary for the actual deadline to be explicitly mentioned when we want to delete a deadline. A change in the `validateArguments` might be needed for this. So ideally, a command to delete a deadline would look like `delete -a "Akutami" -m "Kaisen" -d`.

Edit: This is just me thinking about implementation details, but a suggestion I have about implementing this is in the `DELETE` switch case we update the options for `-d` to not require an argument and add a `isDelete` flag in `validateArguments` to bypass the check for a quoted argument following `-d`. Might not be the cleanest implementation. To be further discussed!